kloppy/domain/models/common.py

```diff
@@ -940,28 +941,92 @@ def to_records(
         else:
             return iterator
 
+    def to_dict(
```

Comment:
Detail: I'm wondering whether the `to_dict` method name could be potentially confusing. Intuitively, without looking at the actual implementation, I didn't expect this method to return a `dict` where each value corresponds to a list of values by default. For instance, the `pandas.DataFrame` class also offers a `to_dict` method which shows the same behavior when passing the `list` argument to the `orient` parameter.